packages/manager/modules/telecom-universe-components/src/telecom/voip/service/voip-service.factory.js

```diff
@@ -69,6 +69,26 @@ export default () => {
       return this.description || this.serviceName;
     }
 
+    /**
+     *  @ngdoc method
```

Comment:
I think it must be a function not a method.
method don't allow return statement.

```suggestion
     *  @ngdoc function
```